Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.h

```diff
@@ -42,6 +42,7 @@ class CLuaElementDefs : public CLuaDefs
     LUA_DECLARE(GetElementDimension);
     LUA_DECLARE(GetElementZoneName);
     LUA_DECLARE(GetElementBoundingBox);
+    LUA_DECLARE_OOP(GetElementBoundingBox);// Probably wrong, tell me how to declare the OOP version of this.
```

Comment:
This is ok. You should remove `LUA_DECLARE(GetElementBoundingBox)` as `LUA_DECLARE_OOP(GetElementBoundingBox)` automatically does both.

Reply:
Thanks, I'll remove it.